### ISSUE TITLE
`StaticNetlink.update_or_create` PyCharm warnings

### ISSUE THREAD

**Issue by stdedos** [gabstopper/smc-python#29]
On `smc-python>=0.6.1`: (i.e. also `smc-python-0.6.2rc1`)

```python
StaticNetlink.update_or_create(
        name=name,
        gateway=Router(router_name),
        network=[Network(network_NAME)],
        provider_name='ISP #{:d}',
        probe_address=[THE_OTHER_SIDE],
        comment=comment
).add_category([TAG_NAME])
```

![image](https://user-images.githubusercontent.com/133706/43207124-f72c72a6-902f-11e8-9cbe-a07b544bab42.png)

et al. (`Expected 'dict', got 'x' instead`)

but it executes correctly

**Comment by gabstopper** [comment]
Looks like pycharm is grabbing the docstring and interpreting that. I've removed the docstring referencing kwargs as a dict. Will be addressed on 0.6.2-rc.2

**Comment by stdedos** [comment]
Yes, that is the cause (tested locally)

**Comment by gabstopper** [comment]
Resolved in current develop branch, >=0.6.2-rc.2
